interpreter/cli.py

```diff
@@ -148,7 +155,22 @@ def cli(interpreter):
     # This way, when folks hit interpreter --local, they get the same experience as before.
     
     rprint('', Markdown("**Open Interpreter** will use `Code Llama` for local execution. Use your arrow keys to set up the model."), '')
-        
+    if interpreter.offline:
+      models = {}
+      # get all models in the models folder
+      user_data_dir = appdirs.user_data_dir("Open Interpreter")
```

Comment:
doenst import appdirs

Reply:
good catch, I think this pr maybe massively out of date now and may be in need of a complete rework to be intergrated into the current code base, I still think there is something to be said for ensuring open-interpreter can still run when there is no internet connection